### PR TITLE
AAAAAAAAAA

### PR DESCRIPTION
**Pull request by adolf-hitler2**

### CODE REVIEW

**Comment by baraa-almasri** [comment]
replace prinr w/ print!

File: lol.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
 #!/usr/bin/python3
 
 prinr("cyka blyat")
```
Comment (baraa-almasri) [inline lol.py:3]: replace prinr w/ print

**Review by baraa-almasri — CHANGES_REQUESTED** [review]

File: lol.py (hunk 1 above)
Comment (adolf-hitler2) [reply lol.py:3]: ok

**Review by adolf-hitler2 — COMMENTED** [review]
done

**Pull request closed without merging**